src/components/program/ProgramSubscriptionPlanAdminCard.tsx

```diff
@@ -66,6 +82,9 @@ const ProgramSubscriptionPlanAdminCard: React.FC<{
     appId,
     `ProgramPlan_${programPlan?.id}`,
   )
+  const { loading: loadingById, data: dataById } = useQuery(GET_CARD_TITLE_BY_ID, {
+    variables: { id: programPlan.card_id },
```

Comment:
convert `cardId `

Reply:
we can set the skip condition into `useQuery`
```ts
  const { loading: loadingById, data: dataById } = useQuery(GET_CARD_TITLE_BY_ID, {
    variables: { id: programPlan.card_id },
    skip: !!programPlan.card_id
```
FYI: https://www.apollographql.com/docs/react/api/react/hooks/#usequery-parameters-options-skip